lms/templates/header/user_dropdown.html

```diff
@@ -0,0 +1,40 @@
+## mako
+<%page expression_filter="h"/>
+<%namespace name='static' file='static_content.html'/>
+
+<%!
+from django.conf import settings
+from django.urls import reverse
+from django.utils.translation import ugettext as _
+
+from openedx.core.djangoapps.user_api.accounts.utils import retrieve_last_sitewide_block_completed
+%>
+
+<%
+## This template should not use the target student's details when masquerading, see TNL-4895
+self.real_user = getattr(user, 'real_user', user)
+username = self.real_user.username
+resume_block = retrieve_last_sitewide_block_completed(self.real_user)
+email = self.real_user.email
+xpro_base_url = settings.XPRO_BASE_URL
```

Comment:
@blarghmatey I have added `XPRO_BASE_URL` in my `lms.env.json` but it was not available in settings. After adding it to my settings `lms/env/devstack.py`, it is working now. 
Question: Is adding new var in `lms.env.json` is sufficient.

Reply:
I believe there is code in `lms/env/aws.py` or `lms/env/production.py` to pull in extra values from the `lms.env.json`, but I could be mistaken. I'm not sure if it's present in the devstack settings. To verify you can try running the app and specifying the settings to be either `aws` or `production` so that those respective settings files get loaded in place of `devstack.py`.

If there is not code to handle loading those extra values then I guess we would need to add it ourselves.

Reply:
@blarghmatey Thanks for the explanation. I can see that @shaidar has already added `XPRO_BASE_URL` at https://github.com/mitodl/salt-ops/commit/ea9e13a203ad5696f8167f7d0dc5146f87536808#diff-a47bb65f0755a70156468ed33e9f1f60R62. Do I still need to do anything here ?

Reply:
I can see, `XPRO_BASE_URL` is used in my other PR which is merged and deployed.